CHANGELOG.md

```diff
@@ -3,7 +3,8 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### API breaking changes
 
-* None.
+* The `key` parameter of `objectForPrimaryKey(_:, key:)`/ `dynamicObjectForPrimaryKey(_:, key:)`
```

Comment:
I don't think this is an API breaking change, since non-optionals are implicitly converted to optionals in Swift.

Also, the convention for referring to APIs would make these `Realm.objectForPrimaryKey(_:key:)` and `Realm.dynamicObjectForPrimaryKey(_:key:)` (add `Realm.` and remove the comma).